lib/Kaltura/LibraryController.php

```diff
@@ -138,16 +138,16 @@ public function sendtoeditorAction() {
 			$uiConfId    = KalturaHelpers::getRequestPostParam( 'uiConfId' );
 			$playerRatio = KalturaHelpers::getRequestPostParam( 'playerRatio' );
 			$hoveringControls = KalturaHelpers::getRequestPostParam( 'hoveringControls' );
-
-			$varMakeResponsive = KalturaHelpers::getRequestPostParam( 'makeResponsive' );
-			$isResponsive = !empty($varMakeResponsive);
+			$isResponsive = ! empty( $width ) && $width === '100%';
```

Comment:
`! empty( $width )` is not required here. `$width` will always be initialized. It can be null/empty string/0, but you compare it with `'100%'`, which is fine.